cloud/pkg/edgecontroller/controller/downstream.go

```diff
@@ -133,6 +134,25 @@ func (dc *DownstreamController) syncConfigMap() {
 				continue // continue to next select
 			}
 
+			// It is used to inform the local cluster gateway changes to its neighbors
+			if configMap.Name == "cluster-gateway-config" {
```

Comment:
add this string to a variable to be shared with other places

Reply:
Added